tests/integration/components/sl-calendar-test.js

```diff
@@ -0,0 +1,773 @@
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-calendar .datepicker-switch' ).text().trim(),
+        'Septiembre ' + currentYear,
+        'Current month in Spanish is set correctly in the view'
+    );
+});
+
+test( 'Action fires when day is clicked', function( assert ) {
+
+    assert.expect( 1 );
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    const currentMonth = 9;
+    this.set( 'currentMonth', currentMonth );
+
+    this.set( 'content', testContent );
+
+    this.render( hbs`
+        {{sl-calendar
+            action='testAction'
+            content=content
+            currentYear=currentYear
+            currentMonth=currentMonth
+        }}
+    ` );
+
+    this.on( 'testAction', () => {
+        assert.ok( 'The test action was called' );
+    });
+
+    this.$( '.sl-calendar .active' ).click();
+});
+
+test( 'Action passes through expected objects in content array', function( assert ) {
+
+    assert.expect( 8 );
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    const currentMonth = 9;
+    this.set( 'currentMonth', currentMonth );
+
+    this.set( 'content', multipleTestContent );
+
+    this.render( hbs`
+        {{sl-calendar
+            action='testAction'
+            content=content
+            currentYear=currentYear
+            currentMonth=currentMonth
+        }}
+    ` );
+
+    this.on( 'testAction', ( dateContent ) => {
+
+        // This group of asserts verifies the multiple events on Sep. 17th
+        if ( 'Event 1 Today!' === dateContent[ 0 ].label ) {
+            assert.equal(
+                dateContent[ 0 ].date.toString(),
+                'Sat Sep 17 2022 00:00:00 GMT-0500 (CDT)',
+                'The date property was passed through'
+            );
+
+            assert.equal(
+                dateContent[ 0 ].label,
+                'Event 1 Today!',
+                'The label property was passed through'
+            );
+
+            assert.equal(
+                dateContent[ 1 ].date.toString(),
+                'Sat Sep 17 2022 00:00:00 GMT-0500 (CDT)',
+                'The date property was passed through'
+            );
+
+            assert.equal(
+                dateContent[ 1 ].label,
+                'Event 2 Today!',
+                'The label property was passed through'
+            );
+
+            assert.equal(
+                dateContent[ 2 ].date.toString(),
+                'Sat Sep 17 2022 00:00:00 GMT-0500 (CDT)',
+                'The date property was passed through'
+            );
+
+            assert.equal(
+                dateContent[ 2 ].label,
+                'Event 3 Today!',
+                'The label property was passed through'
+            );
+        // This group of asserts verifies the event on Sep. 20th
+        } else {
+            assert.equal(
+                dateContent[ 0 ].date.toString(),
+                'Tue Sep 20 2022 00:00:00 GMT-0500 (CDT)',
```

Comment:
While this assertion is specifically checking for `GTM -0500 (CDT)` a timezone was never specified in `multipleTestContent`.  So while this test will pass in the Central timezone, it will not in any others.  Case in point: https://travis-ci.org/softlayer/sl-ember-components/jobs/77575444 is the TravisCI build for this PR - it is failing due to the date actually being `Sat Sep 17 2022 00:00:00 GMT+0000 (UTC)`.